src/match/tasks.py

```diff
@@ -1,58 +1,56 @@
 import datetime
-import logging
-import os
-import sys
 
-from celery import Celery
-from celery.signals import after_setup_logger
-from django.utils import timezone
+from celery import shared_task
+from django.contrib.auth.models import User
+from django.db.models import Q
 from django_celery_beat.models import IntervalSchedule
 from django_celery_beat.models import PeriodicTask
 from match import match_status
 from match.controllers.create_match_controller import CreateMatchController
 from match.models import Match
-from person.models import Person
+from pear_algorithm.src.main import main as pear_algorithm
 
-# Get Pear algorithm
-current_dir = os.path.dirname(os.path.abspath(__file__))
-submodule_path = f"{current_dir}/../../pear-algorithm/src"
-sys.path.insert(0, submodule_path)
-from main import main
+# PROOF OF CONCEPT
 
-app = Celery()
+schedule, _ = IntervalSchedule.objects.get_or_create(
+    every=1,
+    period=IntervalSchedule.MINUTES,
+)
 
-# https://www.distributedpython.com/2018/08/28/celery-logging/
-logger = logging.getLogger(__name__)
 
+@shared_task
+def update_noah():
+    noah = User.objects.get(person__net_id="njs99")
+    noah.last_name = "Solomon " + datetime.datetime.now().strftime("%m/%d/%Y, %H:%M:%S")
+    noah.save()
+    return "Saved Noah"
 
-@after_setup_logger.connect
-def setup_loggers(logger, *args, **kwargs):
```

Comment:
yeah this logger thing was unnecessary you can just print

Reply:
it did provide some different levels of information though but i don't think we'll need that level of depth for our matcher